package/MDAnalysis/coordinates/TNG.py

```diff
@@ -0,0 +1,437 @@
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+TNG trajectory files --- :mod:`MDAnalysis.coordinates.TNG`
+==========================================================
+
+
+The TNG format is a format used in `GROMACS`_ for storage of trajectory and
+topology information. The TNG format allows a wide range of compression
+algorithms and unlike the compressed XTC format can also store velocities
+forces in addition to positions.
+
+The classes in this module are based on the `pytng`_ package for reading TNG files
+The reader is directed to the `documentation`_ for further reading about how pytng
+works under the hood.
+
+In addition to particle-dependent trajectory information like positions,
+forces and velocities, the TNG format can store trajectory metadata and 
+other arbitrary time dependent data. Additional information can range from
+the virial and pressure components to the molecular topology of the system.
+This is enabled by a block based system in which binary flags indicate the
+presence or absence of various data blocks. The structure of a TNG file is 
+provided  in the TNG `specification`_. The TNG `paper`_ and the
+pytng `documentation`_ are also good resources. The user is encouraged to read
+the full list of `blocks`_ to understand the full power of the TNG format.
+
+
+
+Notes
+-----
+Currently there is only a Reader for TNG files and no writer. This will depend
+on upstream changes in pytng. Additionally, it is not currently possible to
+read the molecular topology from a TNG file.
+
+There are also some limitations to reading TNG files with pytng.
+Currently we do not allow data to be read *off stride*. In essence this
+means that all of the critical trajectory data (positions, box, velocities
+(if present), forces (if present)) must share the same stride in trajectory
+integrator steps. These critical blocks in the TNG file are henceforth called
+*special blocks*. Optional blocks (all blocks that are not special blocks)
+will not be read if they do not share an integrator step with, or are not
+divisible by the shared integrator step of the special blocks.
+
+
+.. Links
+
+.. _GROMACS:
+    https://www.gromacs.org/
+.. _pytng:
+    https://github.com/MDAnalysis/pytng
+.. _documentation:
+    https://www.mdanalysis.org/pytng/
+.. _blocks:
+    https://www.mdanalysis.org/pytng/documentation_pages/Blocks.html
+.. _specification:
+    https://gitlab.com/hugomacdermott/tng/-/blob/master/Trajectoryformatspecification.mk
+.. _paper:
+    https://onlinelibrary.wiley.com/doi/10.1002/jcc.23495
+
+"""
+
+import numpy as np
```

Comment:
Please sort all these imports as per PEP8, I feel like @PicoCentauri was using some tool/scheme to deal with this.

Reply:
fixed using `isort` (for reference)